flask/main.py

```diff
@@ -103,7 +105,24 @@ def cronUpdate():
     #Cron authentication through post request data
     req_data = request.get_json()
     responsedata = ''
-    if (req_data['user'] != 'user') or (req_data['pass'] != 'pass'):
+
+    #Get secret values from Datastore environment variables
+    client = datastore.Client()
+    query = client.query(kind = 'env_vars')
+    entity = query.fetch()
+
+    secretindex = 0
```

Comment:
This is nice, I'd improve this logic here to get the values by key I was testing and looks like this will work out.

```
secrets = list(entity)[0]
slackurl = secrets.get('slack_api')
passw = secrets.get('pass')
user = secrets.get('user')
```

Or just leave them in the dict secrets and access them directly in the code.

Reply:
Ah thanks, I thought I was overlooking a simpler method, forgot list typecasting.